webextensions/api/scripting.json

```diff
@@ -0,0 +1,351 @@
+{
+  "webextensions": {
+    "api": {
+      "scripting": {
+        "InjectionTarget": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/scripting/InjectionTarget",
+            "support": {
+              "chrome": {
+                "version_added": "88"
+              },
+              "edge": {
+                "version_added": "88"
+              },
+              "firefox": {
+                "version_added": "101"
+              },
+              "firefox_android": {
+                "version_added": "101"
+              },
+              "opera": {
+                "version_added": "74"
+              },
+              "safari": {
+                "version_added": "15.4",
+                "notes": "Supported in Manifest V2 and V3 as a parameter of <code>executeScript</code>, <code>insertCSS</code>, and <code>removeCSS</code>."
```

Comment:
This note is unnecessary. Let's remove it.

`InjectionTarget` is always a parameter of the mentioned APIs. I previously commented about MV2/MV3 because your previous version of the BCD entry mentioned something about MV3 compat in every note.

Reply:
Done